src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -314,31 +336,22 @@ class ReportingConfigForm extends React.Component {
           />
         )}
         <div className="col">
-          <ValidationFormGroup
-            for="enterpriseCustomerCatalogs"
-            helpText="The catalogs that should be included in the report. No selection means all catalogs will be included."
-          >
-            <label htmlFor="enterpriseCustomerCatalogs">Enterprise Customer Catalogs</label>
-            <Input
-              type="select"
-              id="enterpriseCustomerCatalogs"
-              name="enterpriseCustomerCatalogUuids"
-              multiple
-              defaultValue={selectedCatalogs}
-              options={
-                availableCatalogs.map(item => ({
-                  value: item.uuid,
-                  label: `Catalog "${item.title}" with UUID "${item.uuid}"`,
-                }))
-              }
-            />
-          </ValidationFormGroup>
+          {this.renderSelect({
+            key: 'enterpriseCustomerCatalogUuids',
+            helpText: 'The catalogs that should be included in the report. No selection means all catalogs will be included.',
+            label: 'Enterprise Customer Catalogs',
+            options: availableCatalogs.map(item => ({
+              value: item.uuid,
+              label: `Catalog "${item.title}" with UUID "${item.uuid}"`,
+            })),
+            multiple: true,
+            defaultValue: selectedCatalogs,
+          })}
         </div>
         <div className="row justify-content-between align-items-center form-group">
-          <ValidationFormGroup
-            for="submitButton"
-            invalidMessage="There was an error submitting, please try again."
-            invalid={submitState === SUBMIT_STATES.ERROR}
+          <Form.Group
+            submitButton="submitButton"
```

Comment:
`Form.Group` doesn't accept a `submitButton` prop? Is this supposed to be `controlId`?

Reply:
my bad